scripts/meta_analysis.py

```diff
@@ -73,7 +73,14 @@ def n_meta( studies : List[Tuple['Study','VariantData']] ) -> Tuple:
     beta_meta=sum_betas/sum_weights
     
     #TODO se
-    return ( beta_meta, None, max(sys.float_info.min * sys.float_info.epsilon, 2 * scipy.stats.norm.sf( abs( sum( effs_size ) ) / math.sqrt(tot_size) )), effs_size_org, weights) if len(effs_size)==len(studies) else None
+    return (
+        beta_meta,
+        None,
+        max(sys.float_info.min * sys.float_info.epsilon, 2 * scipy.stats.norm.sf( abs( sum( effs_size ) ) / math.sqrt(tot_size) )),
+        -(scipy.stats.norm.logsf( abs( sum( effs_size ) ) / math.sqrt(tot_size) ) + math.log(2)) / math.log(10),
```

Comment:
there is log10 directly in match also

Reply:
Yes, but the `scipy.stats.norm.logsf` spits out natural log so have to convert that to log10.